ci_scripts/ci_post_clone.sh

```diff
@@ -42,6 +42,7 @@ else
   pip3 install aqtinstall
   aqt install-qt -O /Volumes/workspace/repository/qt_ios mac desktop $QTVERSION -m qtwebsockets qt5compat
   aqt install-qt -O /Volumes/workspace/repository/qt_ios mac ios $QTVERSION -m qtwebsockets qt5compat
+  mv /Volumes/workspace/repository/qt_ios/$QTVERSION/macos /Volumes/workspace/repository/qt_ios/$QTVERSION/clang_64
```

Comment:
Well, in theory this is not needed because we pass `macos` here: `  ./scripts/macos/apple_compile.sh ios -q /Volumes/workspace/repository/qt_ios/$QTVERSION/macos/bin`

Reply:
The issue is that we need this at the correct place because the iOS `qmake` is referencing the macos `qmake`.